lib/puppet/provider/consul_acl/default.rb

```diff
@@ -53,18 +46,11 @@ def self.list_resources
     nacls
   end
 
-  def self.prefetch(resources)
```

Comment:
Hmm. I understand why you have to do this.
Let me think a bit about possible other solutions.

I'm a big fan of self.instances and prefetch. I think there might be a better way to gracefully degrade..